src/components/Todo/TodoStyles.ts

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+import { Grid, Text } from "@mantine/core";
+
+export const StyledText = styled(Text)``;
+
+export const TodoWrapper = styled(Grid)`
+  width: 100%;
+  height: 4rem;
+  flex-wrap: nowrap;
+
+  &.mantine-Grid-root .mantine-Col-root {
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    margin: 0;
+
+    .mantine-Text-root {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
+    }
+  }
+
+  &.mantine-Grid-root > div:nth-child(2) {
+    border-top: 2px solid #eee;
+    border-bottom: 2px solid #eee;
+    padding: 0 1rem 0 1rem;
```

Comment:
`padding: 0 1rem;` udělá přesně to samé. A ještě lépe `padding-inline: 1rem` nastaví odsazení jen ze stran pokud nepotřebuješ padding shora a zdola nastavovat. Obdobně `padding-block` nastavuje shora a zdola. Margin má pak stejné možnosti. Nezmiňoval bych, ale v kódu projektu to najdeš celkem často :)